lib/cerner/oauth1a/protocol.rb

```diff
@@ -57,10 +57,18 @@ def self.parse_authorization_header(value)
         value = value.strip
         return params unless value.size > 6 && value[0..5].casecmp?('OAuth ')
 
-        value.scan(/([^,\s=]*)=\"([^\"]*)\"/).each do |pair|
-          k = URI.decode_www_form_component(pair[0])
-          v = URI.decode_www_form_component(pair[1])
-          params[k.to_sym] = v
+        # trim off 'OAuth ' prefix
```

Comment:
Should we also add an example of token to the method doc? https://github.com/cerner/cerner-oauth1a/blob/716d43405cd16a8a0dd56ffa91a1dbfc70a52cdf/lib/cerner/oauth1a/protocol.rb#L48-L50

Reply:
Added at b9aecce.